straxen/plugins/event_processing.py

```diff
@@ -714,11 +716,11 @@ def compute(self, events):
 
             # corrected s2 with s2 xy map only, i.e. no elife correction
             # this is for s2-only events which don't have drift time info
-            cs2_top_wo_elifecorr = (events[f'{peak_type}s2_area'] * events[f'{peak_type}s2_area_fraction_top'] / 
-                                    self.s2_map(s2_positions, map_name=s2_top_map_name))
+            cs2_top_wo_elifecorr = (events[f'{peak_type}s2_area'] * events[f'{peak_type}s2_area_fraction_top'] /
+                                    self.s2_xy_map(s2_positions, map_name=s2_top_map_name))
             cs2_bottom_wo_elifecorr = (events[f'{peak_type}s2_area'] *
                                        (1 - events[f'{peak_type}s2_area_fraction_top']) /
-                                       self.s2_map(s2_positions, map_name=s2_bottom_map_name))
+                                       self.s2_xy_map(s2_positions, map_name=s2_bottom_map_name))
             result[f"{peak_type}cs2_wo_elifecorr"] = (cs2_top_wo_elifecorr + cs2_bottom_wo_elifecorr)
```

Comment:
I think this guy is now a bit misleading as cs2 != cs2_wo_elifecorr * elife_correction. At least I find this quite confusing.

Reply:
Maybe call it `cs2_only_pos_corr`?

Reply:
Otherwise no idea either

Reply:
Nothing is actually changing here except for `cs2` and the addition of this `cs2_wo_timecorr` variable, so that would be my preference personally. But interested to hear what others think. @jingqiangye @l-althueser thoughts? Should we change the variable names to be more clear, but also break all the analyses downstream that use these variables like `cs2_wo_elifecorr`?

Reply:
Hey @ershockley  my opinions will be:
* keep `cs2_wo_elifecorr` as 1) it's used in the [cS2AFT cut](https://github.com/XENONnT/cutax/blob/0ed85580c6a26e5ce4c1841d2d360bd3d8c83e7e/cutax/cuts/s2_aft.py#L50) and 2) it'll be useful for S2-only events which don't have elife correction
* save the time correction factor for SE_EE, instead of saving `cs2_wo_timecorr`, as it's not directly used in any analysis. But one can always get it by cS2 * time_correction_factor. This is the same thought as @l-althueser 

To make `cs2_wo_elifecorr` meaningful, we need to apply the time correction factor there. If people agree, I can make a commit towards this direction.

Reply:
@jingqiangye `cs2_wo_timecorr` is equivalent to the current cs2, so I think it is definitely useful to have it to compare things as we work on this new correction. Is there a reason that saving the correction factor is better? To me it seems like an unnecessary middle step but maybe I'm missing something. 

And yes good point on the order the corrections are applied. It makes most sense to apply elife last.

Reply:
@ershockley I think that `cs2_wo_timecorr` is just a temporary field when we are not sure of the time correction performance. But the time correction factor will be there anyway even if the correction performance is validated. I assume that when we know the correction is good, we don't want `cs2_wo_timecorr` to still be there to complicate things. For the performance validation, we can still easily compute `cs2_wo_timecorr` using the time correction factor.

However, I don't have a strong opinion about this and won't insist if people feel that the other way is better. What I would insist on is to keep `cs2_wo_elifecorr`, but this seems not a problem :)

Reply:
> I think that cs2_wo_timecorr is just a temporary field when we are not sure of the time correction performance. But the time correction factor will be there anyway even if the correction performance is validated

Why do we need the correction factor there long term?